target/linux/ath79/dts/ar9132_buffalo_wzr-hp-g300nh.dtsi

```diff
@@ -0,0 +1,249 @@
+			linux,code = <BTN_5>;
+			gpios = <&gpio2 2 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+
+		movie_off {
+			label = "movie_off";
+			linux,code = <BTN_3>;
+			gpios = <&gpio2 3 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+
+		usb {
+			label = "usb";
+			linux,code = <BTN_2>;
+			gpios = <&gpio2 5 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		router_auto {
+			label = "router_auto";
+			linux,code = <BTN_6>;
+			gpios = <&gpio2 6 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+
+		movie_on {
+			label = "movie_on";
+			linux,code = <BTN_4>;
+			gpios = <&gpio2 7 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+	};
+
+	flash@1e000000 {
+		compatible = "cfi-flash";
+		reg = <0x1e000000 0x2000000>;
+		bank-width = <2>;
+		device-width = <2>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0000000 0x0040000>;
+				read-only;
+			};
+
+			partition@40000 {
+				label = "u-boot-env";
+				reg = <0x0040000 0x0020000>;
+				read-only;
+			};
+
+			partition@60000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x0060000 0x1f60000>;
+			};
+
+			partition@1fc0000 {
+				label = "user_property";
+				reg = <0x1fc0000 0x0020000>;
+				read-only;
+			};
+
+			art: partition@1fe0000 {
+				label = "art";
+				reg = <0x1fe0000 0x020000>;
+				read-only;
+			};
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		usb {
+			label = "blue:usb";
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port>;
+			linux,default-trigger = "usbport";
+		};
+
+		led_diag: diag {
+			label = "red:diag";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		wireless {
+			label = "green:wireless";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		led_security: security {
+			label = "amber:security";
+			gpios = <&gpio 17 GPIO_ACTIVE_LOW>;
+		};
+
+		router {
+			label = "green:router";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	switch: switch {
+		status = "disabled";
+
+		gpio-sda = <&gpio 19 GPIO_ACTIVE_HIGH>;
+		gpio-sck = <&gpio 20 GPIO_ACTIVE_HIGH>;
+		mii-bus = <&mdio0>;
+
+		mdio-bus {
+			status = "okay";
+
+			#address-cells = <1>;
+			#size-cells = <0>;
+			phy-mask = <0x10>;
+
+			phy4: ethernet-phy@4 {
+				reg = <4>;
+				phy-mode = "rgmii";
+			};
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+};
+
+&eth0 {
+	status = "disabled";
+
+	phy-mode = "rgmii";
+	mtd-mac-address = <&art 0x1120c>;
+
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&mdio1 {
+	status = "okay";
+};
+
+&eth1 {
+	status = "disabled";
+
+	compatible = "qca,ar9130-eth", "syscon";
+	reg = <0x1a000000 0x200
+		0x18070004 0x4>;
+
+	pll-reg = <0x4 0x18 22>;
+	pll-handle = <&pll>;
+
+	phy-mode = "rgmii";
+	phy-handle = <&phy4>;
+	resets = <&rst 13>;
+	reset-names = "mac";
+	qca,mac-idx = <1>;
+	mtd-mac-address = <&art 0x1120c>;
```

Comment:
Not sure whether we already discussed this, but ar71xx uses increment +1 for the MAC address on eth1.

Reply:
Actually we did discuss it. :)

In the stockfirmware all interfaces have the same MAC address and it was decided to not change them here either.

Reply:
Okay, fine, thanks. Please add a note about that to the commit message, so nobody changes it back it 1 year due to the same reason.

Reply:
Done.